osu.Game/Screens/Edit/EditorBeatmap.cs

```diff
@@ -96,6 +96,7 @@ public EditorBeatmap(IBeatmap playableBeatmap, ISkin beatmapSkin = null, Beatmap
             PlayableBeatmap.ControlPointInfo = ConvertControlPoints(PlayableBeatmap.ControlPointInfo);
 
             this.beatmapInfo = beatmapInfo ?? playableBeatmap.BeatmapInfo;
+            if (this.beatmapInfo.TimelineZoom <= 0) this.beatmapInfo.TimelineZoom = 1;
```

Comment:
Don't know about that. Are you aware that there are limits to zooming in the timeline? Surely zero isn't in that range, but it still ended up being used, which is not correct behaviour.

Check how zooming is set up in `Timeline`, that should be enough to tell you what's actually wrong.